src/components/AccountActivityDataCellAex141TransferEvent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <value-hash-ellipsed
+    :hash="activityPayload.contractId"
+    :link-to="`/contracts/${activityPayload.contractId}`"/>
```

Comment:
```suggestion
  <value-hash-ellipsed
    :hash="activityPayload.recipientId"
    :link-to="`/accounts/${activityPayload.recipientId}`"/>
```

Should be the recipient, it's "Transferred to".

Reply:
Make sense, fixed